recipes/fgbio/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version="0.2.1b" %}
-{% set tag="2af6747" %}
+{% set version="0.4.0" %}
+{% set tag="a9445b4" %}
```

Comment:
Why not use the tag rather than the commit hash?

Reply:
May be a good idea to check how the meta.yaml was setup when it referenced a prior release.

Reply:
To be honest, this is the first time I've poked at [conda build recipes](https://conda.io/docs/user-guide/tasks/build-packages/recipe.html); I wanted the 0.4.0 release via conda as it really simplifies reproducibility for me, so I made the minimal changes necessary. I've made another commit, so take a look there please. Thanks for the prompt release, by the way.